### DISCUSSION TITLE
alpha-beta 作业中公式的量纲

### DISCUSSION THREAD

**Discussion by liuhanqing-phy** [physics-data/faq#146] [Q&A] [answered]
公式左边的量纲是J/m，右边是J·m^2，是哪里缺了常数吗？
![image](https://user-images.githubusercontent.com/303360/133568950-25f4ea2d-b0de-403e-8395-1f0f23fdf745.png)

**Comment by heroxbd** [comment, marked as answer]
怎么解决的？量纲对上了吗？

alpha-beta 中公式的量纲对上了。

**Reply by liuhanqing-phy** [reply]
嗯，N是带单位“每立方米”的，刚开始忘算了

**Reply by heroxbd** [reply]
有道理，好的。

**Comment by Harry-Chen** [comment]
请勿随意将问题改成”已解决“等标题！这是一种非常糟糕的习惯。
